lnd.go

```diff
@@ -197,7 +197,7 @@ type rpcListeners func() ([]*ListenerWithSignal, func(), error)
 // is received on the shutdownChan at which point everything is shut down again.
 func Main(cfg *Config, lisCfg ListenerCfg, shutdownChan <-chan struct{}) error {
 	defer func() {
-		ltndLog.Info("Shutdown complete")
+		ltndLog.Infof("Shutdown complete")
```

Comment:
This isn't the same as `fmt.Printf()` vs. `fmt.Println()`. This change has no effect at all. Did you mean to append an extra `\n`?

Reply:
yeah, i'm still learning Go and I misread the usage of the `....f` logging utilities.

guess the easiest thing to do here would be `Info("Shutdown complete\n")`?

Reply:
just a random question: why doesn't this method add a line break by default?